### PR TITLE
[2/X] Add POST ability to persistent storage

### PR DESCRIPTION
**Pull request by dash102**
Comments get loaded in chronological order from DataStore

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -29,25 +32,33 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<String> comments = new ArrayList<>();
+    Query query = new Query(dataKind).addSort("timestamp", SortDirection.ASCENDING);;
+    PreparedQuery results = datastore.prepare(query);
+    
+    for (Entity entity : results.asIterable()) {
+      String name = (String) entity.getProperty("name");
+      String comment = (String) entity.getProperty("comment");
+      long timestamp = (long) entity.getProperty("timestamp");
+      comments.add(getCommentStatement(name, comment, timestamp));
```
Comment (hlz03) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: you can create a function like buildComment(Entity);

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -58,6 +69,13 @@ private String convertToJson(List<String> data) {
     return new Gson().toJson(data);
   }
 
+  /**
+   * @return the desired comment format to display
+   */
+  private String getCommentStatement(String name, String comment, long timestamp) {
+    return "at " + timestamp + ", "  + name + " said: " + comment;
```
Comment (hlz03) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:76]: prefer String.format.

And it's not required to have a java doc for private method.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (dash102) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (dash102) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -48,23 +59,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
   private static String convertToJson(List<String> data) {
     return new Gson().toJson(data);
   }
 
-  /**
-   * @return the request parameter, or the default value if the parameter
-   *         was not specified by the client
-   */
+  private static String getCommentStatement(String name, String comment, long timestamp) {
+    return String.format("at %d, %s said: %s", timestamp, name, comment);
```
Comment (hlz03) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:78]: you can just return this in buildComment
another way is you implement this as a toString() method in Comment class, and in buildComment you build a Comment object and return comment.toString().

**Pull request merged**